plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -134,11 +162,12 @@ def calc_max_ber_wait_time(self, min_threshold):
         min_bits = float(format(float(min_threshold), '.0e').replace('-', ''))
         min_sec_to_wait = min_bits / min_port_rate
         return min_sec_to_wait
+
     def is_out_of_operating_conf(self, port_name):
-        port_telemetry = self.ports_data.get(port_name)
-        if not port_telemetry:
+        port_obj = self.ports_data.get(port_name)
+        if not port_obj:
```

Comment:
INHO - need to print something that you did not find port_obj and skip ... is it normal?

Reply:
done